v2/js/screens/loginscreen.js

```diff
@@ -270,7 +277,10 @@ const LoginScreen = {
 					}
 				}
 				else if (this.index.currentIndex === 2) { // password
-					this.index.currentIndex++;
+					this.details.password = this.$refs.passwordInput.passwordText;
+					if (this.details.password.length >= 4) {
```

Comment:
Why not using the constant created before?
What about the minimum server size mentioned here : https://github.com/llaske/sugarizer/issues/1594#issuecomment-2052471106 ?

Reply:
Coming to that, I also wanted to know like, if you could elaborate on what do you want to be done in min server size
I didn't get it completely